CONTRIBUTING.md

```diff
@@ -193,6 +193,12 @@ test('returns a mocked response', async ({ loadExample, fetch }) => {
 
 ##### Running all browser tests
 
+Make sure Playwright chromium has been installed before running browser tests.
+
+```sh
+pnpm playwright install chromium
```

Comment:
This is kind of implied by using `playwright` (it won't run without browsers) and isn't really specific to MSW. I would rather not include this step here. With it here, we need to make sure it reflects any possible future changes to `playwright`, its browser usage, or its CLI.